server/mongodb/actions/IssueManager.ts

```diff
@@ -29,3 +29,17 @@ export async function getIssueById(id: ObjectId): Promise<EntityDoc> {
   await connectToDB();
   return IssueDocument.findById(id);
 }
+
+export async function completeIssueById(id: ObjectId): Promise<EntityDoc> {
+  await connectToDB();
+  return IssueDocument.findByIdAndUpdate(
+    id,
+    { status: "RESOLVED", dateCompleted: new Date() },
```

Comment:
It may be better to use the luxon library and do DateTime.now(). I'm assuming this will work just fine as well, as you probably can pass in a Date into a DateTime object.